### PR TITLE
Preannouncement of DesktopVirtualization Upcoming Breaking Changes

### PR DESCRIPTION
**Pull request by msJinLei**
## Description

Pick the upcoming breaking change from https://github.com/Azure/azure-powershell/pull/26249 and do some revisions.

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by alec-baird — APPROVED** [review]

File: documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- DaysOfWeek
+### The following properties have changed from struct to string:
+- SystemDataLastModifiedByType
+- SystemDataCreatedByType
+- SystemDataCreatedAt
+- SystemDataLastModifiedAt
+- SystemDataLastModifiedBy
+- SystemDataCreatedBy
+### Array replaced by List due to the generated tool (autorest.powershell) upgrading
+- Details can be found [Breaking Changes in Generated Modules Due to AutoRest.PowerShell Upgrade from v3 to v4](https://github.com/Azure/azure-powershell/blob/main/documentation/breaking-changes/breaking-changes-in-generated-modules-due-to-codegen-tool-upgrade-from-v3-to-v4.md#list-replaces-array-in-generated-c-classes)
```
Comment (msJinLei) [inline documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md:78]: https://learn.microsoft.com/en-us/powershell/azure/breaking-changes-generated-modules?view=azps-12.4.0

File: documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- StartupBehavior
+- SessionHostLoadBalancingAlogrithm
+- StopHostsWhen
+- RemoteApplicationType
+- SessionHostComponentUpdateType
+- LoadBalancerType
+- PersonalDesktopAssignmentType
+- PreferredAppGroupType
+- DaysOfWeek
+### The following properties have changed from struct to string:
```
Comment (isra-fel) [inline documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md:70]: Are "enum to string" and "struct to string" the same category?

File: documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# Upcoming breaking changes in Azure PowerShell
+
+## Az.DesktopVirtualization
+
+The following cmdlets were affected this release:
```
Comment (isra-fel) [inline documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md:5]: Are these cause by API version update? Can we be more specific?

File: documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md (hunk 2 above)
Comment (msJinLei) [reply documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md]: > Are "enum to string" and "struct to string" the same category?

should be in same category

File: documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md (hunk 3 above)
Comment (msJinLei) [reply documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md]: > Are these cause by API version update? Can we be more specific?

Talked to xiaogang and comfirm that all of them are caused by V4 upgrading

**Review by isra-fel — APPROVED** [review]
I directly committed on top of your branch. The main changes include:

1. Title
2. links - when pasting links from learn, remove the locale (en-us) and the version (azps-12.4.0)
3. Reduced level of cascading

**Pull request merged**